setup.py

```diff
@@ -30,9 +30,8 @@
         "python-gitlab>=1.11.0,<1.12.0",
         "semver~=2.13",
         "python-terraform>=0.10.0,<0.11.0",
-        # memory leak after boto3 1.24.72, https://github.com/boto/boto3/issues/3614
-        "boto3==1.24.71",
-        "botocore==1.27.71",
+        "boto3==1.26.134",
+        "botocore==1.29.134",
```

Comment:
This will be removed once https://github.com/app-sre/qontract-reconcile/pull/3529 is merged.